tests/integration/cmd_watch_test.go

```diff
@@ -50,6 +50,15 @@ var _ = Describe("odo watch command tests", func() {
 		})
 	})
 
+	Context("when executing watch with wrong app name", func() {
+                It("should fail", func() {
+                        helper.CopyExample(filepath.Join("source", "nodejs"), context)
+                        helper.CmdShouldPass("odo", "component", "create", "nodejs", "--project", project, "--context", context)
+                        output := helper.CmdShouldFail("odo", "watch", "--app", "dummy")
```

Comment:
you need to use `--context` here for running the command

Reply:
yes, will update